src/components/workspace/WorkspaceTreeView/WorkspaceTreeView.svelte

```diff
@@ -0,0 +1,268 @@
+    if (contextMenuNode?.type !== WorkspaceContentType.Directory) {
+      targetPath = targetPath.split(PATH_DELIMITER).slice(0, -1).join(PATH_DELIMITER);
+    }
+    dispatch('importFile', targetPath);
+  }
+
+  function onCopyFileLocation() {
+    let targetPath = contextMenuNode?.fullPath ?? '';
+    dispatch('copyFileLocation', targetPath);
+  }
+
+  function onMoveToWorkspace() {
+    let targetPath = contextMenuNode?.fullPath ?? '';
+    dispatch('moveToWorkspace', targetPath);
+  }
+</script>
+
+<div class="h-auto pt-1">
+  {#if enableContextMenu}
+    <ContextMenuInternal bind:this={contextMenu} on:hide={onContextMenuHide}>
+      <ContextMenu.Group>
+        <ContextMenu.Item size="sm" on:click={onRenameNode} aria-label="Rename">
+          <div
+            class="flex gap-1"
+            use:permissionHandler={{
+              hasPermission: hasEditPermission,
+              permissionError: 'You do not have permission to edit this workspace',
+            }}
+          >
+            <PencilLine size={16} />
+            Rename
+          </div>
+        </ContextMenu.Item>
+        <ContextMenu.Item size="sm" on:click={onMoveNode} aria-label="Move">
+          <div
+            class="flex gap-1"
+            use:permissionHandler={{
+              hasPermission: hasEditPermission,
+              permissionError: 'You do not have permission to edit this workspace',
+            }}
+          >
+            <FolderOutput size={16} />
+            Move
+          </div>
+        </ContextMenu.Item>
+        <ContextMenu.Item size="sm" on:click={onDeleteNode} aria-label="Delete">
+          <div
+            class="flex gap-1"
+            use:permissionHandler={{
+              hasPermission: hasDeletePermission,
+              permissionError: 'You do not have permission to delete this workspace',
+            }}
+          >
+            <Trash2 size={16} />
+            Delete
+          </div>
+        </ContextMenu.Item>
+      </ContextMenu.Group>
+      <ContextMenu.Separator />
+      <ContextMenu.Item size="sm" on:click={onCopyFileLocation} aria-label="Copy Link to">
+        <div
+          class="flex gap-1"
+          use:permissionHandler={{
+            hasPermission: hasEditPermission,
+            permissionError: 'You do not have permission to edit this workspace',
+          }}
+        >
+          <Copy size={16} /> Copy Link to {contextMenuNode?.type === WorkspaceContentType.Directory
+            ? 'Directory'
+            : 'File'}
+        </div>
+      </ContextMenu.Item>
+      <ContextMenu.Separator />
+      <ContextMenu.Item size="sm" on:click={onMoveToWorkspace} aria-label="Move to Workspace">
+        <div class="flex gap-1">
+          <FileOutput size={16} /> Move to Workspace
+        </div>
+      </ContextMenu.Item>
+      <ContextMenu.Separator />
+      <ContextMenu.Group>
+        <ContextMenu.Item size="sm" on:click={onNewSequence} aria-label="New Sequence">
```

Comment:
Would be nice if clicking new sequence at the top would default to the user's current folder like the menu below it does, especially since the down arrow menu below is a little more hidden.

<img width="573" height="200" alt="Image" src="https://github.com/user-attachments/assets/167cfe34-d35f-4919-bda0-9d545ae86c7a" />